rasa/cli/arguments/run.py

```diff
@@ -1,9 +1,29 @@
+import argparse
+
+from rasa.cli.arguments.default_arguments import add_model_param
 from rasa.core import constants
 
 
-def add_run_arguments(parser):
+def set_run_arguments(parser: argparse.ArgumentParser):
+    add_server_arguments(parser)
+    add_model_param(parser)
+
+
+def set_run_action_arguments(parser: argparse.ArgumentParser):
+    import rasa_core_sdk.cli.arguments as sdk
+
+    sdk.add_endpoint_arguments(parser)
+
+    parser.add_argument(
+        "--actions",
+        type=str,
+        default="actions",
+        help="Name of action package to be loaded.",
+    )
+
+
+def add_server_arguments(parser: argparse.ArgumentParser):
     parser.add_argument(
-        "-o",
```

Comment:
do we need to remove this, is there another arg using this?

Reply:
It is used for `log-file` and `output`. To avoid the confusion, I just removed the `-o` from both. However, we could reintroduce it to one of them.

Reply:
ok, no makes sense, should go in the migration notes as well

Reply:
yes, already done :)